package.json

```diff
@@ -9,11 +9,20 @@
     "name": "AlCalzone",
     "email": "d.griesel@gmx.net"
   },
-  "main": "build/index.js",
-  "types": "build/index.d.ts",
+  "main": "build/cjs/index.js",
+  "types": "build/esm/index.d.ts",
+  "module": "build/esm/index.js",
+  "exports": {
```

Comment:
In matter.js we ended in this structure https://github.com/project-chip/matter.js/blob/main/packages/matter.js/package.json#L58-L67 ... there was a point for this additional types/default substructure

Reply:
From my experience so far this is not needed as the esm types should be just fine for cjs as well.